app/assets/sass/modules/_multi_select.scss

```diff
@@ -0,0 +1,102 @@
+
+
+div.multi-select {
+  position: relative;
+  @extend %contain-floats;
+  width:100%;
```

Comment:
Pedantic, but there should be spaces between the `:` and the value i.e. `width: 100%`. There's a lot of occurrences in this file. There is a sass-lint file in the repo, if you use a plugin it will highlight these and you can prolly run an autofix thing

Reply:
fixed